data/src/main/java/com/ssafy/data/boiler/model/BoilerItemDto.kt

```diff
@@ -0,0 +1,23 @@
+package com.ssafy.data.boiler.model
+
+import com.google.gson.annotations.SerializedName
+
+data class BoilerItemDto(
+    @SerializedName("기업명")
+    val enterprise: String?, // 기업명
```

Comment:
json의 기업명이라는 필드의 값을 enterprise에 매핑해주기 위해 이렇게 작성한 것으로 보입니다.
다만 json의 필드명은 보통 영문으로 smallCamelCase로 작성해주거나 snake_case로 작성해주는 것이 일반적이므로
```suggestion
    val enterprise: String?, // 기업명
```
다음과 같이 작성하고 json의 필드명을 enterprise로 동일하게 나중에 백엔드에서 응답주게 구현하면 될 것 같습니다.

Reply:
필드명을 CamelCase로 작성하도록 하겠습니다.